front/src/modules/study/components/AddOrEditStudyModal.tsx

```diff
@@ -413,7 +413,7 @@ const AddOrEditStudyModal = ({ editionMode, study, scenarios }: AddOrEditStudyMo
           <div className="col-lg-4">
             <InputSNCF
               id="studyInputBudget"
-              type="number"
+              type="text"
```

Comment:
Can we keep `number` still? This is useful for accessibility, on-screen keyboards, etc.

Reply:
But with `text` the user can also type letters and special characters? So we can keep `number`, and add some additional validation?